### PR TITLE
[NET-716] Fix category update error on type builder

### PR DESCRIPTION
**Pull request by markheumueller**
[NET-716](https://nethinks.atlassian.net/browse/NET-716)

### CODE REVIEW

File: app/src/app/framework/type/type-builder/type-builder.component.ts [hunk 1]
```diff
@@ -124,15 +124,19 @@ export class TypeBuilderComponent implements OnInit {
...
-                console.log('Type id removed from category');
+            // Remove from old category
+            if (this.basicStep.originalCategoryID) {
+              this.categoryService.getCategory(this.basicStep.originalCategoryID).subscribe((category: CmdbCategory) => {
+                const index = category.types.indexOf(this.typeInstance.public_id, 0);
+                if (index > -1) {
+                  category.types.splice(index, 1);
+                }
+                this.categoryService.updateCategory(category).subscribe(() => {
+                  console.log('Type id removed from category');
```
Comment (doblanken) [inline app/src/app/framework/type/type-builder/type-builder.component.ts:135]: The log seems rather redundant

**Review by doblanken — COMMENTED** [review]
looks good all together, that log however doesn't seem of much use to me.

File: app/src/app/framework/type/type-builder/type-builder.component.ts (hunk 1 above)
Comment (markheumueller) [reply app/src/app/framework/type/type-builder/type-builder.component.ts:135]: Yes, old leftover. This can actually be deleted.

**Review by doblanken — APPROVED** [review]
LGTM

**Pull request merged**